aboutMe/AboutMe.md

```diff
@@ -0,0 +1,51 @@
+# Resume
+
+Content
+
+[TOC]
+
+---
+
+## Basic information
+
+### Name
+
+Jamal Guyo
+
+### Title
+
+Software Developer
```

Comment:
You can put the title inline with the title above 
**Title:** Software developer

Reply:
resolved